frontend/src/Components/RecipeOfTheDay.jsx

```diff
@@ -0,0 +1,33 @@
+// src/components/RecipeOfTheDay.jsx
+import React, { useEffect, useState } from 'react';
+
+function RecipeOfTheDay() {
+  const [recipe, setRecipe] = useState(null);
+
+  useEffect(() => {
+    // Fetch a random recipe from the API
+    fetch('/api/recipes/random')
+      .then((response) => response.json())
+      .then((data) => setRecipe(data))
```

Comment:
@aditiverma-21 you have used the random api here but where did you create this API??